sources/Debate/src/main/java/com/example/Debate/dto/ActivityDto.java

```diff
@@ -0,0 +1,12 @@
+package com.example.Debate.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
```

Comment:
IMO dto's should be immutable. Instead of @Getter and @Setter, a @Value annotation will generate the required code - getter, required args constructor and toString, equals and hashcode implementations. The fields in class will be set to private final